<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PprzMap</class>
 <widget class="QWidget" name="PprzMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="srtm_button">
       <property name="text">
        <string>SRTM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="srtm_label">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
        </font>
       </property>
       <property name="text">
        <string>SRTM label</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="reference_combobox">
       <item>
        <property name="text">
         <string>WGS84</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WGS84 (sexagesimal)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pos_label">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
        </font>
       </property>
       <property name="text">
        <string>pos_label</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="MapWidget" name="map"/>
   </item>
   <item>
    <widget class="QPushButton" name="config_button">
     <property name="text">
      <string>Planificacion</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapWidget</class>
   <extends>QGraphicsView</extends>
   <header>mapwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
